### DISCUSSION TITLE
2 factor organization

### DISCUSSION THREAD

**Discussion by coupal** [community/community#3648] [Mobile] [answered]
Hi! 
Are you planning on supporting 2 FA organization soon?

I'm currently unable to open PR from the @zendesk org. Since it requires 2 factor.

Thanks,
JM

**Comment by coupal** [comment, marked as answer]
I needed to **logout and login again**. 
It would be great to see in the organization list that ability to authorized it. (I was previously unable to see this one)

**Reply by xxksa2010** [reply]
او

**Reply by xxksa2010** [reply]
G
